data/MBS/data.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "MBS",
+  "symbol": "MBS",
+  "decimals": 18,
+  "description": "MonkeyLeague is the next-gen casual esports game empowering players to create, play, compete, and earn.",
+  "website": "https://monkeyleague.io",
+  "twitter": "@TheMonkeyLeague",
+  "tokens": {
+    "ethereum": {
+      "address": "0x4Bbf46E7c31eDc1F33aF4d57F0C4a0f13398427c"
```

Comment:
If you change the top-level name to "MonkeyLeague" you can add the following here to eliminate the validation error around mismatching name:
```
override: {
  name: "MonkeyLeague"
}
```

Reply:
I tried it the other way around with overriding the goerli tokens with the top-level `MBS` name. will that work?

Reply:
yeah I think so.